pkg/nsx/services/vpc/vpc.go

```diff
@@ -55,3 +55,7 @@ func InitializeVPC(service common.Service) (*VPCService, error) {
 
 	return VPCService, nil
 }
+
+func (s *VPCService) GetVPCsByNamespace(namespace string) []model.Vpc {
```

Comment:
This function is a method of VPCService, but from caller perspective, we have to hold a 
vpc service firstly and call this function. It's good that we have this function inside VPCService. we also need to consider how to expose vpcservice to users.

Reply:
you are correct, i have raised the question in our slack channel to discuss how would we handle inter-service communication.